docs/tutorials/coupled_bucket.jl

```diff
@@ -0,0 +1,152 @@
+# # Setting up a Coupled Simulation
+
+# For more information about the bucket model,
+# please see [the bucket model tutorial](https://clima.github.io/ClimaLSM.jl/dev/generated/bucket_tutorial/).
+
+# This tutorial shows how to set up a simulation for a coupled simulation. More detail for coupled runs can be
+# found in the ClimaCoupler.jl [documentation](https://clima.github.io/ClimaCoupler.jl/dev/). In
+# preparation for understanding this tutorial, we recommend also reading the [intro to multi-component models tutorial](https://clima.github.io/ClimaLSM.jl/dev/generated/LSM_single_column_tutorial/) as well as being familiar
+# with multiple dispatch programming in Julia.
+
+# # Background
+
+# Recall that in order to drive the system in standalone mode,
```

Comment:
I wonder about removing the discussion about setting up the standalone bucket in this tutorial, or at least shortening it a bit.